### DISCUSSION TITLE
Option to execute statement at cursor position

### DISCUSSION THREAD

**Discussion by m0rphZ3us** [mtxr/vscode-sqltools#1369] [Ideas]
Hey, I like the tools, so I don't have to use my phpmyadmin anymore. I would love to have the option to excute a statement at cursor position like in SQLTools for mssql.

**Comment by gjsjohnmurray** [comment]
Have you seen https://vscode-sqltools.mteixeira.dev/en/features/executing-queries already? You should be able to select the query text and run it this way.

**Reply by m0rphZ3us** [reply]
Hi John, thanks for your reply. I got it working now.
